packages/plugin-dotnet-scripting/index.js

```diff
@@ -0,0 +1,31 @@
+import {
+  FAKE_NUGET,
+  NET_SCRIPT_FILE,
+  NET_SCRIPT_NUGET,
+} from '@octolinker/helper-grammar-regex-collection';
+import nugetResolver from '@octolinker/resolver-nuget';
+import relativeFile from '@octolinker/resolver-relative-file';
+
+export default {
+  name: 'DotNetScripting',
+
+  resolve(path, [target]) {
+    // this reference doesn't resolve to anything
+    if (target === './.fake/build.fsx/intellisense.fsx') {
+      return [];
```

Comment:
Just to share some insights, you can use `undefined` as a return value as well. Just saying, no need to change it.

Reply:
@stefanbuck I just tried `return;` and `return undefined;` and both give me an error. I guess that's why I was using an empty array here.

```
Uncaught (in promise) TypeError: Cannot read property 'registry' of undefined
    at app.js:41806
    at Array.map (<anonymous>)
    at app.js:41795
    at app.js:40329
    at Array.map (<anonymous>)
    at run (app.js:40322)
    at app.js:40355
    at gitHubInjection (app.js:15027)
    at init (app.js:40350)
```

The error points back to this line https://github.com/OctoLinker/OctoLinker/blob/dbaaaa5d845ed83198043201150753539210ee09/packages/helper-normalise-resolver-results/index.js#L67

Reply:
Ohh sorry for pointing you into the wrong direction. I guess there is some value in having a consistent return type 😉